.github/workflows/dev-release.yml

```diff
@@ -44,5 +44,6 @@ jobs:
       - name: Copy files to release directory
         run: |-
           gcloud storage rm -r gs://seqr-luigi/releases/dev/latest/ || echo 'No latest release'
-          gcloud storage cp v03_pipeline/bin/* gs://seqr-luigi/releases/dev/latest/
+          gcloud storage cp v03_pipeline/bin gs://seqr-luigi/releases/dev/latest/bin/
+          gcloud storage cp v03_pipeline/var/vep_config gs://seqr-luigi/releases/dev/latest/var/vep_config
```

Comment:
Changed the structure of the deploy artifact a bit so the vep configs are deployed and accessible by the dataproc init.